src/cockpit/bridge.py

```diff
@@ -280,7 +280,7 @@ def main(*, beipack: bool = False) -> None:
         print(f'Version: {__version__}\nProtocol: 1')
         return
     elif args.bridges:
-        print(json.dumps(Packages().get_bridge_configs(), indent=2))
+        print(json.dumps([config.to_json() for config in Packages().get_bridge_configs()], indent=2))
```

Comment:
This is debugging, more or less, so how about just `config.__dict__`?

Reply:
SGTM